data/strategies/StrategyMakerWBTCDAIDelegate.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyMakerWBTCDAIDelegate",
+  "description": "Locks wBTC at MakerDAO via WBTC-A to mint DAI. This newly minted DAI is then deposited into the v2 DAI yVault.",
```

Comment:
Do we want to say something about the debt ratio? or may tiny bit longer explanation of this strat?